src/transformers/__init__.py

```diff
@@ -91,6 +91,7 @@
     "debug_utils": [],
     "dependency_versions_check": [],
     "dependency_versions_table": [],
+    "doc_utils": [],
```

Comment:
Additions here are for the repo consistency checks to pass. When we settle on names and objects to be moved, each submodule will contain the objects of the public init it defines.